src/index.js

```diff
@@ -1,7 +1,7 @@
 const app = require("./app");
 require("dotenv").config();
 
-const port = process.env.PORT || 5000;
+const port = process.env.PORT || 6000;
```

Comment:
Why changing it here ?

Reply:
Because I had another project with this port on wich i didn't can change it.